crates/gateway-client/src/lib.rs

```diff
@@ -1610,22 +1610,22 @@ mod tests {
             let actual = get_latest_version(&Client::integration()).await.unwrap();
             assert_eq!(
                 actual,
-                StarknetVersion::new(0, 11, 1),
+                StarknetVersion::new(0, 11, 2),
```

Comment:
Wait did integration already get bumped? Or are you just doing so pre-emptively?

Reply:
It did: https://github.com/eqlabs/pathfinder/actions/runs/5066679745/jobs/9096799280